bundles/AdminBundle/Resources/views/Admin/Login/login.html.php

```diff
@@ -27,16 +27,16 @@
 
 
 <div id="loginform">
-    <form method="post" action="<?= $view->router()->path('pimcore_admin_login_check', ['perspective' => strip_tags($view->request()->getParameter('perspective'))]) ?>" autocomplete="off">
+    <form method="post" action="<?= $view->router()->path('pimcore_admin_login_check', ['perspective' => strip_tags($view->request()->getParameter('perspective'))]) ?>">
 
         <?php if ($this->error) { ?>
             <div class="text error">
                 <?= $this->translate($this->error) ?>
             </div>
         <?php } ?>
 
-        <input type="text" name="username" placeholder="<?= $this->translate("Username"); ?>" required autofocus/>
-        <input type="password" name="password" placeholder="<?= $this->translate("Password"); ?>" required/>
+        <input type="text" name="username" autocomplete="username" placeholder="<?= $this->translate("Username"); ?>" required autofocus>
+        <input type="password" name="password" placeholder="<?= $this->translate("Password"); ?>" required>
```

Comment:
Maybe add `autocomplete="current-password"` here?

```suggestion
        <input type="password" name="password" autocomplete="current-password" placeholder="<?= $this->translate("Password"); ?>" required>
```

Reply:
Could have sworn I did